package.json

```diff
@@ -11,6 +11,7 @@
       "default": "./dist/main.js"
     },
     "./dist/": "./dist/",
+    "./tokens": "./dist/esm/src/tokens.scss.js",
```

Comment:
This will create a js entry which is interesting  approach.
There might be an issue in terms of css loading order in case this will be loaded after other files were already loaded.

If we want to do the same as today - we actually need to create a css output “bundle”

Reply:
Won't it inject styles at the top as we are using `style-inject`?

Reply:
that's how it works, but I'm not sure if the order of loading is guaranteed, just need to check it